core/unit_test/tools/TestCInterface.c

```diff
@@ -0,0 +1,2 @@
+#include "../../src/impl/Kokkos_Profiling_C_Interface.h"
```

Comment:
I would likely be happier if this could just be
```suggestion
#include <impl/Kokkos_Profiling_C_Interface.h>
```
and you pass an appropriate include path to `gcc`